[Issue #1196] Change default behaviour of new:project (#1263)
boostercloud/booster: docs/chapters/02_getting-started.md

ℹ Project generated!
```

> > [!TIP] If you prefer to create the project with default parameters, you can run the command as `boost new:project booster-blog --providerPackageName <package-name>`. The package name of the provider is the only required parameter in project creatio. The default

Reviewer: ```suggestion
> [!TIP] If you prefer to create the project with default parameters, you can run the command `boost new:project booster-blog --providerPackageName <package-name>`. The package name of the provider is the only required parameter and can be any `npm` package that implements Booster's provider interfaces. The framework provides the following: 

* `@boostercloud/framework-provider-aws`: Runs your app in AWS Lambda and stores the events, entity snapshots, and read models in DynamoDB.
* `@boostercloud/framework-provider-azure`: Runs your app in Azure Functions and stores the events, entity snapshots, and read models in CosmosDB.
* `@boostercloud/framework-provider-kubernetes` (Experimental): This package is a proof of concept that runs a Booster application on a Kubernetes cluster and Redis as the database.

The default
```

Reviewer 2: I think that the DynamoDB, Cosmos, Lambda, etc... is too low-level for the getting started section. I'd remove that, specially with the K8s provider, given it is deprecated now 😅

ℹ Project generated!
```

> [!TIP] If you prefer to create the project with default parameters, you can run the command as `boost new:project booster-blog --providerPackageName <package-name>`. The package name of the provider is the only required parameter and can be any `npm` package that implements Booster's provider interfaces. The framework provides the following: 
>

> > - `@boostercloud/framework-provider-aws`: Runs your app in AWS Lambda and stores the events and entity snapshots.
> > - `@boostercloud/framework-provider-azure`: Runs your app in Azure Functions and stores the events and entity snapshots.
> > - `@boostercloud/framework-provider-kubernetes` (Experimental): This package is a proof of concept that runs a Booster application on a Kubernetes cluster.

Reviewer: If we don't want to mention the specific technologies, this might be easier for a beginner:

```suggestion
> - `@boostercloud/framework-provider-aws`: Runs your app in AWS.
> - `@boostercloud/framework-provider-azure`: Runs your app in Azure.
> - `@boostercloud/framework-provider-kubernetes` (Experimental): Proof of concept that shows how to run your app on a Kubernetes cluster.
```

What do you think @hannahbrooks @NickSeagull ?

Reviewer: That's explained in more depth in other sections of the docs, so I think there is no need to repeat it here. I initially suggested it as part of the explanation of what's stored in the database, but as we're not mentioning it anymore, I'm not sure that this extra bit provides valuable information. What do you think?

Author: no i agree, i think it's probably best to keep the getting started section as simple as possible!
